src/views/Blog.vue

```diff
@@ -74,6 +83,33 @@ export default {
   },
   methods: {
     get,
+    jsPDF,
+
+    async getPDF() {
+      this.$refs.pdfContent.innerHTML = this.$refs.blog.innerHTML;
+      const page = this.$refs.pdfContent;
```

Comment:
This whole function could potentially fail due to many reasons
- Library configuration errors
- Missing refs or code related issues
Such methods should always have a `try/catch` block and in the `catch` notify the user about the error with a notification.